src/kOS/Module/kOSProcessor.cs

```diff
@@ -82,7 +95,7 @@ public void Activate()
             OpenWindow();
         }
 
-        [KSPField(isPersistant = true, guiName = "Required Power", guiActive = true)]
+        [KSPField(isPersistant = true, guiName = "kOS Required Power", guiActive = true, guiActiveEditor = true)]
```

Comment:
> So, when the game is actually running, it is the real current power requirement.

In both places where the code is calculating this `RequiredPower` value for display, both here AND down at line 653 below where it is calculating it during flight, it's using this same formula.  It never uses the real runtime drain calculation based on `shared.Cpu.InstructionsThisUpdate`.  It uses `Safehouse.Config.InstructionsPerUpdate` in both places.

That was the difference I was talking about.  A script that is doing a `wait until foo.` is yielding most of its IPU.

I made my comment about the tooltip because I thought this was deliberate and you *wanted* to display the same calculation in the ediitor as during runtime.  But your comment that you wanted to display the actual value at runtime makes me think there may be a cut and paste error here because I see InstructionsPerUpdate being used the same way in both places.